lib/internal/webstreams/util.js

```diff
@@ -129,21 +131,25 @@ function transferArrayBuffer(buffer) {
   return res;
 }
 
-function isArrayBufferDetached(buffer) {
+function isDetachedBuffer(buffer) {
+  assert(isArrayBuffer(buffer));
```

Comment:
Is this useful? I think the error thrown by V8 enough (and clearer).
```suggestion
```

Reply:
On second thought, seemingly it can be deleted in the current version of this PR. Fixed.